app/views/users/registration_closed.html.erb

```diff
@@ -2,11 +2,10 @@
   <div class="container">
     <div class="details">
 
-      <h1 class="title is-1">Hacktoberfest has ended</h1>
+      <h1 class="title is-1">See you next year!</h1>
       <p>
-        Registrations are now closed.
-
-        Hope to see you next year!
+        Hacktoberfest 2020 has ended and registration is now closed. We hope to
+        see you next year for another round of open source celebration!
       </p>
 
       <br/>
```

Comment:
`class="line-br"` is a holdover from last year I believe, and was removed from the other, similar partials in the app.

Reply:
Replaced with `underline-div`, which is used on the error pages